vrp-pragmatic/src/format/solution/writer.rs

```diff
@@ -40,6 +40,9 @@ pub trait PragmaticSolution<W: Write> {
 
     /// Serializes solution in pragmatic geo json format.
     fn write_geo_json(&self, problem: &Problem, writer: &mut BufWriter<W>) -> Result<(), String>;
+
+    /// Serializes solution in, both, pragmatic and geo json format.
+    fn write_geo_json_and_pragmatic_json(&self, problem: &Problem, writer: &mut BufWriter<W>) -> Result<(), String>;
```

Comment:
I would change a bit the name to something like `write_pragmatic_and_geo_json` . Reasons:
- `pragmatic` model somehow dominates here and its method is defined first, so better to put it on the first place
- avoid using json word twice